views/home.dt

```diff
@@ -27,6 +27,18 @@ block body
 	- else
 		p Welcome to DUB, the D package registry. The following list shows all available packages:
 
+	//New Section
```

Comment:
please don't add comments that refer to old/new things, this comment will be outdated immediately on the next change and isn't helping anyone. I would just suggest removing this line since the `h2` describes it already